api/prisma/schema.prisma

```diff
@@ -374,7 +376,7 @@ model Jurisdictions {
   publicUrl                     String                   @default("") @map("public_url")
   emailFromAddress              String?                  @map("email_from_address")
   rentalAssistanceDefault       String                   @map("rental_assistance_default")
-  whatToExpect                  String                   @default("") @map("what_to_expect")
+  whatToExpect                  String                   @default("Applicants will be contacted by the property agent in rank order until vacancies are filled. All of the information that you have provided will be verified and your eligibility confirmed. Your application will be removed from the waitlist if you have made any fraudulent statements. If we cannot verify a housing preference that you have claimed, you will not receive the preference but will not be otherwise penalized. Should your application be chosen, be prepared to fill out a more detailed application and provide required supporting documents.") @map("what_to_expect")
```

Comment:
This was missed in previous PR. It was updated in a migration but not in the schema file so whenever you generate a new migration it tried to set it back to an empty string.